Gemfile

```
@@ -4,7 +4,7 @@ source 'http://rubygems.org'
 
 gemspec
 
-gem 'rubocop', '~> 0.80.0'
+gem 'rubocop', '~> 1.23.0'
 gem 'rubocop-minitest'
 gem 'rubocop-performance'
 gem 'rubocop-rails'
```

If you're updating to 1.23.0, why not use theforeman-rubocop 0.1.x
```suggestion
gem 'theforeman-rubocop', '~> 0.1.0'
```